content/admin/get-galaxy/index.md

```diff
@@ -61,12 +63,19 @@ cp config/galaxy.yml.sample config/galaxy.yml
 
 ## Galaxy over network
 
-To access Galaxy over the network, modify the `config/galaxy.yml` file by changing the `http` setting. Galaxy will bind to any available network interfaces instead of the localhost if you change it like this:
+To access Galaxy over the network, modify the `config/galaxy.yml` file by changing the `http` setting. Galaxy will bind to any available ne
+work interfaces on port 8080 instead of the localhost if you add the following:
```

Comment:
```suggestion
To access Galaxy over the network, modify the `config/galaxy.yml` file by changing the `http` setting. Galaxy will bind to any available network interfaces on port 8080 instead of the localhost if you add the following:
```